chapter05/provision/hosts.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+NETWORK="$1"
+DOMAIN="$2"
+START_OCTET="$3"
+
+cat <<EOF >/etc/hosts
+127.0.0.1       localhost
+${NETWORK}${START_OCTET}   prometheus${DOMAIN}    prometheus
+${NETWORK}$((START_OCTET+1))   grafana${DOMAIN}       grafana
```

Comment:
There is only the prometheus host in this chapter, it seems. This line and the next need to be removed.